### PR TITLE
Revert method for setting body unscrollable to the one that was working

### PR DESCRIPTION
**Pull request by rogatty**
https://wikia-inc.atlassian.net/browse/HG-457

Nothing new, this code was here before. This causes top bar to hide/show after toggling side nav but it's a lower priority task than menu not being scrollable.

### CODE REVIEW

**Comment by hakubo** [comment]
I remember there was some problem with lightbox, does it work with this implementation

File: front/scripts/main/views/ApplicationView.ts [hunk 1]
```diff
@@ -128,13 +129,22 @@ App.ApplicationView = Em.View.extend({
 
 	actions: {
 		setScrollable: function (): void {
-			var $element = $(this.get('element'));
-			$element.off('scroll touchmove mousewheel', this.preventDefault);
+			Em.$('body')
+				.removeClass('no-scroll')
```
Comment (hakubo) [inline front/scripts/main/views/ApplicationView.ts]: `no-scroll` could be property, so it never diverge by accident

**Comment by rogatty** [comment]
@hakubo thanks for tips. We weren't actually making page unscrollable when opening lightbox. It should be fine now.

**Comment by hakubo** [comment]
Can we test it first?
@tomasznapieralski please do not merge yet.

**Pull request merged**